src/error.js

```diff
@@ -737,7 +737,12 @@ export function reportErrorToAnalytics(error, win) {
       'errorName': error.name,
       'errorMessage': error.message,
     });
-    triggerAnalyticsEvent(getRootElement_(win), 'user-error', vars);
+    triggerAnalyticsEvent(
+      getRootElement_(win),
+      'user-error',
+      vars,
+      /** enableDataVars */ false
```

Comment:
If we flipped it to be false by default, would we have less overall changes?

Reply:
Not really. You would see a lot of changes in individual components where I set the value to `true` then. 
I set the default value to `true` because we agreed to handle `data-vars` by default, and I want to keep it that way.